src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -208,6 +228,22 @@ function BulkEmailForm(props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isScheduled, editor.editMode, editor.isLoading, editor.errorRetrievingData, editor.formComplete]);
 
+  /*
+   This will be checking if there are emails added to the emailLearnersList state
+   if so, we will delete emailRecipients "learners" because that is for all learners
+   if not, we will delete the list-learners from emailRecipients because of we won't use the emails
+  */
+  useEffect(() => {
+    if (emailLearnersList.length && !editor.emailRecipients.includes('list-learners')) {
```

Comment:
Can we use individual-students instead of list-learners? So it's self-explanatory.